libkbfs/bare_root_metadata_v3.go

```diff
@@ -88,16 +88,19 @@ type BareRootMetadataV3 struct {
 	// of writing to the given folder.
 	FinalizedInfo *tlf.HandleExtension `codec:"fi,omitempty"`
 
-	// The sequence number of the global Keybase Merkle tree at the
-	// time this update was created (from the writer's perspective).
-	// This field was added to V3 after it was live for a while, and
-	// older clients that don't know about this field yet might copy
-	// it into new updates via the unknown fields copier. Which means
-	// new MD updates might end up referring to older Merkle roots.
-	// That's ok since this is just a hint anyway, and shouldn't be
-	// fully trusted when checking MD updates against the Merkle tree.
+	// DEPRECATED -- see KBMerkleRoot.Seqno instead.
```

Comment:
> The unknown handler would populate the zero value, which is also defined as unknown, right?

If a new client sees an old MD update, the `KBMerkleSeqNo` field would go into the unknown fields set -- there would be no `KBMerkleSeqNo` to set.  If an old client sees a new MD update, there would be no such field, so it would stay as the zero value, i.e. the defined unknown value.